java/src/hu/holyoil/skeleton/testcases/UfoMoves.java

```diff
@@ -22,7 +25,6 @@ protected void load() {
         Logger.RegisterObject(home, "home: Asteroid");
         Logger.RegisterObject(target, "target: Asteroid");
 
-        home.AddSpaceship(ufo);
```

Comment:
Itt miért lett kiszedve? Enélkül is lefut, mert a remove nem dov exceptiont amikor nem találja az elemet a listában, de ettól még szerintem logikailag hozzá kell adni a home-hoz az ufo-t.

Reply:
Mert a konstruktor már megteszi